descope/api/client.go

```diff
@@ -370,6 +376,7 @@ func (c *Client) DoRequest(method, uriPath string, body io.Reader, options *HTTP
 		bearer = fmt.Sprintf("%s:%s", bearer, pswd)
 	}
 	req.Header.Set(AuthorizationHeaderName, BearerAuthorizationPrefix+bearer)
+	c.addDescopeHeaders(req)
```

Comment:
theoretically we can calculate this once and put it on a private member, right?

Reply:
yes, the heavy part is done only once